src/stores/errors/errorsActions.ts

```diff
@@ -0,0 +1,27 @@
+import { Dispatch } from 'redux';
+
+export enum ErrorsActions {
+  RESSET = '@error/reset',
+  SET = '@set/error',
```

Comment:
the convention is "@{reducerName}/{reducerAction}" for regular actions, and "@error/${reducerName}" for error actions